cg/meta/transfer/external_data.py

```diff
@@ -144,43 +144,51 @@ def add_files_to_bundles(
             LOG.info("Adding path %s to bundle %s in housekeeper" % (path, lims_sample_id))
             self.housekeeper_api.add_file(path=path, version_obj=last_version, tags=HK_FASTQ_TAGS)
 
+    def get_failed_samples(self, fastq_paths_to_add: List[Path]) -> List[Path]:
+        failed_sums: List[Path] = []
+        for file in fastq_paths_to_add:
+            pass_check = self.check_fastq_md5sum(file)
```

Comment:
"pass_check" sounds like boolean, but it is not. It is a Path. Just reading the code it looks like you are adding True here on line 152. But you are not - you are adding a path, which also is not a sum. I think the variable names are very misleading here and should be changed (see my comments on this section from the previous review)